src/envoy/http/service_control/handler_utils.cc

```diff
@@ -153,10 +153,10 @@ void fillLoggedHeader(
     return;
   }
   for (const auto& log_header : log_headers) {
-    auto* entry = headers->get(Envoy::Http::LowerCaseString(log_header));
-    if (entry) {
+    auto entry = headers->get(Envoy::Http::LowerCaseString(log_header));
+    if (!entry.empty()) {
       absl::StrAppend(&info_header_field, log_header, "=",
-                      entry->value().getStringView(), ";");
+                      entry[0]->value().getStringView(), ";");
     }
```

Comment:
We can call `getAllofHeaderAsString` here to make all header values be logged

Reply:
Done